test/export/util.test.ts

```diff
@@ -0,0 +1,80 @@
+// (C) 2019 GoodData Corporation
+import { DashboardExport } from "@gooddata/typings";
+import { isDateFilter, sanitizeDateFilter, sanitizeDateFilters } from "../../src/export/util";
+
+describe("util for dashboard export", () => {
```

Comment:
just minor thing common shortcut name for helpers/Utilities is `Utils` not just `Util` as it represents more functions not just one :)

Reply:
Fixed

Reply:
well I meant to rename whole file :D Not just test description :D

Reply:
Fixed.
`utils` is now in `src/utils/export.ts` :)